_data/de/site.yml

```diff
@@ -4,20 +4,20 @@ navigation:
     title: TC39
     agenda: Bevorstehende Agenda
     notes: Notizen der letzten Sitzung
-    sub-menu: Umschalte das Untermenü für TC39
+    sub-menu: TC39 Untermenü umschalten
```

Comment:
It would probably sound better if we had separate texts for opening and closing a sub-menu, since `umschalten` is a very uncommon term.

`TC39 Untermenü öffnen` and `TC39 Untermenü schließen` would work much better and also give context about the current state of that submenu.